nuxeo-connect-client/src/test/java/org/nuxeo/connect/connector/http/ProxyPacConfigurationTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright 2020 Nuxeo SA (http://nuxeo.com/).
+ * This is unpublished proprietary source code of Nuxeo SA. All rights reserved.
+ * Notice of copyright on this source code does not indicate publication.
+ *
+ * Contributors:
+ *     Gildas Lefevre
+ */
```

Comment:
`nuxeo-connect-client` is public code, we use the standard Nuxeo copyright header with the Apache license for it.